api/app/registration/tests.py

```diff
@@ -150,11 +141,100 @@ def test_create_offer_with_template(self):
 
         self.assertEqual(response.status_code, 201)
         self.assertFalse(data['payment_required'])
-        self.assertFalse(data['travel_award'])
-        self.assertFalse(data['accommodation_award'])
+
+    def test_create_offer_with_grant_tags(self):
```

Comment:
Ah I see this is the test failing with `ValueError: Could not find email template with key offer-awards`. All of our tests use a fake database that starts from nothing and we populate it with exactly the data we want for the test - this gives us maximum control about the test context, rather than relying on some "real" data which may change. This is what the `seed_static_data` methods do in the tests. So to get the email template populated for your test, you'll need to add a line to `seed_static_data`, like `self.add_email_template('offer-awards')`  (eg line 90)